packages/contents/src/contents.ts

```diff
@@ -872,7 +872,7 @@ namespace Private {
     metadata: {
       orig_nbformat: 4,
     },
-    nbformat_minor: 4,
+    nbformat_minor: 5,
```

Comment:
Hmmm... we almost certainly depend on the `@jupyterlab/nbformat` package (if indirectly), so we could replace the  magic numbers with, at least, _someone else's_ magic numbers, a la https://github.com/jupyterlab/jupyterlab/blob/v4.4.0a1/packages/notebook/src/model.ts#L113

```ts
import * as nbformat from '@jupyterlab/nbformat';

// ...
          nbformat: nbformat.MAJOR_VERSION,
          nbformat_minor: nbformat.MINOR_VERSION,
```

Reply:
gah, never mind: that's _still_ going to be `4`:

https://github.com/jupyterlab/jupyterlab/blob/main/packages/nbformat/src/index.ts#L22

Reply:
As we're thinking this goes into `0.5.0`, I think we should probably either:
- hoist the magic numbers up to e.g. `tokens.ts`
- find some other way to verify it against what a compatible install of `jupyter_server` provides